test/extensions/filters/http/ext_proc/ext_proc_integration_test.cc

```diff
@@ -708,6 +708,55 @@ class ExtProcIntegrationTest : public HttpIntegrationTest,
     verifyDownstreamResponse(*response, 200);
   }
 
+  IntegrationStreamDecoderPtr initAndSendDataMxnMode(absl::string_view body_sent,
+                                                     bool end_of_stream) {
+    config_helper_.setBufferLimits(1024, 1024);
+    proto_config_.mutable_processing_mode()->set_request_header_mode(ProcessingMode::SEND);
+    proto_config_.mutable_processing_mode()->set_request_body_mode(
+        ProcessingMode::BIDIRECTIONAL_STREAMED);
+    proto_config_.mutable_processing_mode()->set_request_trailer_mode(ProcessingMode::SEND);
+    proto_config_.mutable_processing_mode()->set_response_header_mode(ProcessingMode::SKIP);
+
+    initializeConfig();
+    HttpIntegrationTest::initialize();
+    codec_client_ = makeHttpConnection(lookupPort("http"));
+    Http::TestRequestHeaderMapImpl default_headers;
+    HttpTestUtility::addDefaultHeaders(default_headers);
+
+    auto encoder_decoder = codec_client_->startRequest(default_headers);
```

Comment:
use an explicit type for this.

Reply:
Why?  It's auto xxx for most testing code calling startRequest().

Reply:
https://google.github.io/styleguide/cppguide.html#Type_deduction : Envoy's style guide defers to Google's public C++ style guide on this issue.

In particular, types that you cannot infer locally from context should be explicit. Exceptions are typically made if types are hard to describe for some reason, and would make the code less readable.

Violations of this style guide in some parts of the code3 do not imply the guide is no longer applicable.

Reply:
done